Tutorials/Editors-Manual/Getting-Started-With-Umbraco/Umbraco-Interface/index.md

```diff
@@ -1,28 +1,58 @@
-page name will load that page’s content into the large content area on the right of the tree view.
+![Environment.png](images/Environment.png)
 
-![section tree content.jpg](images/section tree content.jpg)
+### 1. The three circles
+These circles represent the environments, the first one in the row is **local** environment, the next one on the right from local is to **add new** environments and the last one in the line is your **live** environment.
+clicking these circles will present you with a box below that gives you different options.
 
-![landingPage.jpg](images/landingPage.jpg)
+![Environment.png](images/Utility-links.png)
 
-## Behaviour at Smaller Screen Sizes
+### 2. The three buttons slightly to the right
+The first button in this row opens the overview of the environments on the project, the second button takes you to the [Umbraco Cloud documentation](https://our.umbraco.com/Documentation/Umbraco-Cloud/).
+The last button has three dots, this gives you the following options: 
+- Getting Started
+- Set up a Project
+- Deployment
+- Troubleshooting
 
-By default, on a desktop with a large screen size, the tree view is always visible giving you quick access
-to select and navigate website content. On smaller devices such as laptops and tablets, the tree view will be closed and only opens after you click on one of the navigation icons in the section bar on the left of the screen.
+![Environment.png](images/Search-and-help.png)
 
-After selecting an item within the tree view on a smaller device, it will automatically close to save screen space. If you wish to close the tree view without selecting a page, you can move your mouse out of the tree view, or alternatively on a touch device you can swipe left over the tree view. Below are examples of how the interface will look on a tablet.
 
-![smallScreenDashboard.jpg](images/smallScreenDashboard.jpg)
+### 3. The three buttons in the top right
+ The first button is a search field here you can search for the content that you are looking for.
+The second button is the help button if you click this you will get options for **help**.
+The third button is your **profile**, this will give you the option to edit your profile and password.
 
-![dashboardSlideOut.jpg](images/dashboardSlideout.jpg)
+![Environment.png](images/Two-tabs-left.png)
+
+### 4. The two buttons slightly towards top left
+The first button is **Your Workspace** that takes you to the tab with the circles on it.
+The second button is **Redirect URL Management** this is for redirects on you project.
+
+![Environment.png](images/The-top-bar-right.png)
+
+### 5. The eight buttons in the far top left
+These buttons are the tabs that you will be working under during you work with you project. 
+The first button is **Content** this is where you manage your content.
```

Comment:
This is great! How about putting them into a list? 🤔 I like lists.
What do you think? 😁

Reply:
yeah that is a good idea  😄

Reply:
are you thinking a numbered list or just bullet points as i could see both work 😄